python/mdvtools/server.py

```
@@ -136,6 +137,13 @@ def create_app(
     #     raise Exception(
     #         "Route already exists - can't have two projects with the same name"
     #     )
+
+        # SocketIO upload initialization
+        socketio_manager = initialize_socketio_upload(app)
```

Flake8 (7.2.0)

[error] local variable 'socketio_manager' is assigned to but never used

(F841)